code/modules/xenomorph/acidwell.dm

```diff
@@ -163,19 +163,19 @@
 	for(var/obj/item/explosive/grenade/sticky/sticky_bomb in stepper.contents)
 		if(charges_used >= charges)
 			break
-		if(sticky_bomb.stuck_to == stepper)
-			sticky_bomb.clean_refs()
+		if(sticky_bomb.active)
+			SEND_SIGNAL(sticky_bomb, COMSIG_ITEM_STICKY_CLEAN_REFS)
```

Comment:
you're sending this in two unrelated places, maybe just make a item proc to handle sending the sig?